core/src/main/java/com/opensymphony/xwork2/config/impl/DefaultConfiguration.java

```diff
@@ -125,6 +127,24 @@
  */
 public class DefaultConfiguration implements Configuration {
 
+    public static final Map<String, Object> BOOTSTRAP_CONSTANTS;
```

Comment:
Extracted bootstrap constants into a public static final field for reuse by the default Struts configuration provider and any test code as needed